webapp/src/ts/modules/messages/messages.component.ts

```diff
@@ -152,6 +160,15 @@ export class MessagesComponent implements OnInit, OnDestroy {
     return this.messageContactService
       .getList()
       .then((conversations = []) => {
+        // filter out the lineage level that belongs to the online logged in user
+        if(!this.sessionService.isOnlineOnly()) {
```

Comment:
```suggestion
        if (!this.sessionService.isOnlineOnly()) {
```
As per the code style, we keep spaces round the brackets for `if` statements